core/src/io/anuke/mindustry/ui/fragments/HudFragment.java

```diff
@@ -249,8 +250,15 @@ public void build(Group parent){
             }).top().left();
         });
 
+        Table map_table = new Table();
```

Comment:
You shouldn't create a table variable here. Use the table lambda syntax instead, e.g. `.table(t -> /**stuff*/)`.